aladdin/arg_tools.py

```diff
@@ -0,0 +1,57 @@
+import functools
+import json
+import os
+import subprocess
+import sys
+import pkg_resources
+
+from aladdin.config import PROJECT_ROOT
+
+
+def add_namespace_argument(arg_parser):
+    namespace_def = os.getenv("NAMESPACE", "default")
+    arg_parser.add_argument(
+        "--namespace",
+        "-n",
+        default=namespace_def,
+        help="namespace name, defaults to default current : [{}]".format(namespace_def),
+    )
+
+
+def bash_wrapper():
+    _, *args = sys.argv
+    handler = os.path.join(PROJECT_ROOT, "aladdin.sh")
+    subprocess.run([handler, *args])
+
+
+def container_command(func=None):
```

Comment:
one of the most significant additions in this PR, it redirects a command over to the bash scripts, then the same command is invoked inside the container where it's actually executed